loom/script/serialize/lsBinReader.cpp

```diff
@@ -784,6 +782,9 @@ Assembly *BinReader::loadExecutable(LSLuaState *_vm, utByteArray *byteArray)
     {
         Reference *ref = lmNew(NULL) Reference;
         ref->name     = readPoolString();
+
+        lmAssert(ref->name[0] != 0, "Reference name is empty, try recompiling the executable");
```

Comment:
Hmm, good point, I'm not 100% on it, but I think lsc sgould work, so `rake utility:compileScripts`.